data/publiccloud/log_instance.sh

```diff
@@ -99,7 +99,7 @@ azure_start_log()
 
     inc_unique_counter
     set +e
-    az vm boot-diagnostics get-boot-log --ids $(az vm list -g "$INSTANCE_ID" --query '[].id' -o tsv) > "${OUTPUT_DIR}/$CNT""_boot_log_start.txt" 2>&1
+    az vm boot-diagnostics get-boot-log --ids $INSTANCE_ID > "${OUTPUT_DIR}/$CNT""_boot_log_start.txt" 2>&1
```

Comment:
```suggestion
    az vm boot-diagnostics get-boot-log --ids "$INSTANCE_ID" > "${OUTPUT_DIR}/$CNT""_boot_log_start.txt" 2>&1
```
Only be save with special chars...